Samples/AudioCreation/cs/AudioCreation/AudioCreation.csproj

```diff
@@ -17,6 +17,11 @@
     <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <AppxAutoIncrementPackageRevision>True</AppxAutoIncrementPackageRevision>
+    <AppxBundle>Always</AppxBundle>
+    <AppxBundlePlatforms>x64</AppxBundlePlatforms>
+    <PackageCertificateThumbprint>6FB24FA8A83D2C78CBB1792FA276D83E4D1E7C2D</PackageCertificateThumbprint>
+    <PackageCertificateKeyFile>AudioCreation_TemporaryKey.pfx</PackageCertificateKeyFile>
```

Comment:
Need to remove these updates as we don't check keys into the repo and need VS to regenerate them.